tycho-its/src/test/java/org/eclipse/tycho/test/compiler/CompilerExtraExportsJava17.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Red Hat Inc. and others.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Red Hat Inc. - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.tycho.test.compiler;
+
+import org.apache.maven.it.Verifier;
+import org.eclipse.tycho.test.AbstractTychoIntegrationTest;
+import org.junit.Test;
+
+public class CompilerExtraExportsJava17 extends AbstractTychoIntegrationTest {
```

Comment:
```suggestion
public class CompilerExtraExportsJava17Test extends AbstractTychoIntegrationTest {
```

According to surefire matching rules I think all test classes must end with 'Test'

Reply:
I just copied the `CompilerExtraExports` test class which also does end with `Test`. So maybe I should change the other names too?

Reply:
If you go here:

https://github.com/eclipse/tycho/pull/618/checks?check_run_id=5056657496

expand  the second "raw output" and search for the test/testname it is not found so I think both tests are simply never executed by the CI ... I think because of the name not ending in `Test`

Reply:
Good, then I'm going to make these renames in this PR today.